src/components/PatientCreationPage/StatusPills.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import Container from "@material-ui/core/Container";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import { status } from "../../graphql/queries/templates/patients";
+
+const StatusPills = ({
+  currentStatus,
+  handleChange,
+  error,
+  helperText,
+}: {
+  currentStatus: status | null;
+  handleChange: (e: React.MouseEvent<HTMLElement>, newStatus: status) => any;
+  error?: boolean;
+  helperText?: string;
+}) => {
+  const classes = useCompletePatientButtonStyles();
+  //const statuses = ["Transport", "Release", "Omit", "Delete"];
+  const statuses = [
+    { val: status.ON_SITE, description: "On Site" },
+    { val: status.RELEASED, description: "Released" },
+    { val: status.TRANSPORTED, description: "Transport" },
+  ];
+  return (
+    <Container className={classes.root}>
+      <Typography className={classes.label}>Status:</Typography>
+
+      <ToggleButtonGroup
+        value={currentStatus}
+        exclusive
+        onChange={handleChange}
+        aria-label="text alignment"
+        classes={{
+          groupedHorizontal: classes.buttonGroup,
+        }}
+      >
+        {statuses.map((status) => (
+          <ToggleButton
+            value={status.val}
+            classes={{
+              root: classes.statusPill,
+            }}
+            key={status.toString()}
```

Comment:
`status` is an object. it has to be `key={status.description}` or `key={status.val}`